grammars/toml.cson

```diff
@@ -21,19 +21,50 @@
     'name': 'key.toml'
   }
   {
+    'begin': '"""'
+    'beginCaptures':
+      '0': 'name': 'string.dquote.triple.begin.toml'
+    'end': '"""'
+    'endCaptures':
+      '0': 'name': 'string.dquote.triple.end.toml'
+    'name': 'string.dquote.triple.toml'
```

Comment:
ok, i’ll use `string.quoted.double.block` instead, [like python](https://github.com/atom/language-python/blob/cdb699e7a86fd9f9f84ae561abddb696aad777aa/grammars/python.cson#L1209).

single/double/triple is bullshit since it implies that “triple” means the same as “single” and double (i.e. number of strokes) instead of the number of characters.